opt/start-postgres.sh

```diff
@@ -29,6 +29,11 @@ else
     if ! $running ; then
        echo "" > /home/${SYSTEM_USER}/.postgresql/logfile # empty log files
        rm -vf /home/${SYSTEM_USER}/.postgresql/postmaster.pid
-       ${PSQL_START_CMD}
+       ${PSQL_START_CMD} || cant_start=true
```

Comment:
As @danielhollas suggested, we should distinguish the "migration required" case from other failures and properly handle it here.